templates/article_page.hbs

```diff
@@ -9,17 +9,20 @@
     <section class="article-sidebar">
       {{#if settings.show_articles_in_section}}
         <section class="section-articles collapsible-sidebar">
-        <h3 class="collapsible-sidebar-title sidenav-title">{{t 'articles_in_section'}}</h3>
-        <ul>
-          {{#each section.articles}}
-            <li>
-              <a href="{{url}}" class="sidenav-item {{#is id ../article.id}}current-article{{/is}}">{{title}}</a>
-            </li>
-          {{/each}}
-        </ul>
-        {{#if section.more_articles}}
-          <a href="{{section.url}}" class="article-sidebar-item">{{t 'see_more'}}</a>
-        {{/if}}
+          <button class="collapsible-sidebar-toggle" aria-labelledby="section-articles-title" aria-expanded="false"></button>
+          <h3 id="section-articles-title" class="collapsible-sidebar-title sidenav-title">{{t 'articles_in_section'}}</h3>
```

Comment:
I wonder why we have an empty button with following `h3` explaining it?
Also I'd prefer to always set `type` for buttons.

Reply:
@yara-simargil The reason to have an empty button is to show the chevron only on mobile. It's explained by the `h3` because the purpose of the `collapsible-sidebar` depends on the context (as opposed to `collapsible-nav` that is only meant for navigation).

_i.e.: (toggle) Ticket details, (toggle) Articles in this section, ..._

Good idea with the button `type`. Will update it 👍